web/src/search_pill.ts

```diff
@@ -32,7 +32,7 @@ export type SearchUserPill = {
 
 type SearchPill =
     | {
-          type: "search";
+          type: "search_non_user";
```

Comment:
Would `generic_operator` be a better name for this type? It contrasts with `user_operator` or `keyword` as a better name for keyword search distinct from "search".

Reply:
I like that better! Thanks. Are you suggesting changing the `type` of `SearchUserPill` to `user_operator` too then?